src/alexandria3k/__main__.py

```diff
@@ -156,8 +156,9 @@ def add_subcommand_populate(subparsers):
     parser.add_argument(
         "-s",
         "--sample",
-        # By default the function always returns True
-        default="True",
+        # By default the function always returns a callable function
+        # that returns a tuple with two True values.
+        default="(True, True)",
         type=str,
         help="Python expression to sample the Crossref tables (e.g. random.random() < 0.0002)",
```

Comment:
This now also applies to USPTO, right?  Please also adjust the example as needed.